### ISSUE TITLE
Please tag releases

### ISSUE THREAD

**Issue by leamas** [bdbcat/oesenc_pi#30]
Please tag the release in git. Git tags makes the release level visible for casual users. The missing tags is also also a problem for packagers, see https://github.com/OpenCPN/OpenCPN/issues/1132.

**Comment by so6power** [comment]
Please tag releases, as leamas said, it is really a problem for packagers

**Comment by mdinslage** [comment]
Yes Please, I am getting request all the time to update my Slackware builds, but it is difficult without tagged releases.
